test/remote/gateways/remote_xpay_test.rb

```diff
@@ -2,14 +2,70 @@
 
 class RemoteRapydTest < Test::Unit::TestCase
   def setup
-    @gateway = XpayGateway.new(fixtures(:x_pay))
-    @amount = 200
-    @credit_card = credit_card('4111111111111111')
-    @options = {}
+    @gateway = XpayGateway.new(fixtures(:xpay))
+    @amount = 100
+    @credit_card = credit_card(
+      '5186151650005008',
+      month: 12,
+      year: 2026,
+      first_name: 'Jim',
+      last_name: 'Smith',
+      verification_value: '123',
+      brand: 'master'
+    )
```

Comment:
Nit: Unless the gateway test card checks all of these fields, you can just call the `credit_card` method and give it the `options` you want to change, so: 

```ruby
credit_card({card_number}, {brand: 'master', verification_value: '456'})
```

Reply:
not needed values removed! Thanks for the review!